tests/plugins/plugin.resume.test.js

```diff
@@ -39,17 +39,26 @@ describe('Plugin: Remember Current Page in Cookies', () => {
 });
 
 describe('updateResumeValue', () => {
+  /* Mark 2020-05-02
+    Commenting this out while confirming with Neeraj
+    I'm not sure:
+    - What this test really does
+    - If the instantiation of the prototype counts as a call
+    - If so, why would there be a second call
+    Can't find in production
+    In this branch .toHaveBeenCalledTimes() === 1
   test('starts when BookReaderInit is called', () => {
```

Comment:
Rather than have it commented out, it can be skipped with `test.skip`. This will provide an output at the end of the run indicating how many skipped tests were encountered.

Reply:
done